integrations/tensorflow/e2e/slim_vision_models/slim_vision_model_test.py

```diff
@@ -0,0 +1,88 @@
+# Lint as: python3
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test all vision models from slim lib."""
+
+import posixpath
+
+from absl import app
+from absl import flags
+import numpy as np
+from pyiree.tf.support import tf_test_utils
+from pyiree.tf.support import tf_utils
+import tensorflow as tf
+import tensorflow_hub as hub
+
+FLAGS = flags.FLAGS
+
+# Testing vision models from
+# https://github.com/tensorflow/models/tree/master/research/slim
+# slim models were designed with tf v1 and then coverted to SavedModel
+# they are stored at tensorflow_hub. At least a subset can be viewed here:
+# https://tfhub.dev/s?dataset=imagenet&module-type=image-classification,image-classifier
+flags.DEFINE_string(
+    'model_tag', 'mobilenet_v1_100_224', 'examples of model tag: '
```

Comment:
This seems like an oddly specific default. Could we just have no default? I think the help text should also link to tfhub so people can look up the examples.

Reply:
The tfhub documentation is... lacking. There was some internal documentation but it got deleted for some reason. I don't have anything to link to that I can link to externally.

Reply:
I do have a 'best effort' link to tfhub in a comment below.

Reply:
Yeah I meant the link you've got right above would be good in the help text.

Reply:
Done.